client/src/app/main-menu/main-menu.tsx

```diff
@@ -22,46 +24,41 @@ function MainMenu({setMenuSection}: Props) {
 	const handleDeck = () => setMenuSection('deck')
 
 	return (
-		<div className={css.chooseGame}>
-			<img width="200" height="184" src="/images/tcg1.png" />
-			<div className={css.message}>
-				Welcome <span className={css.playerName}>{playerName}</span>,
```

Comment:
I see we no logner show player name in menu? In that caes remove the `const playerName = useSelector(getPlayerName)` line since we no longer need it (also `useSelector` from imports).